bigtable/tests/filters_integration_test.cc

```diff
@@ -65,7 +65,7 @@ class FilterIntegrationTest : public bigtable::testing::TableIntegrationTest {
 /// Return true if connected to the Cloud Bigtable Emulator.
 bool UsingCloudBigtableEmulator();
 
-}  // namespace anonymous
+}  // namespace
```

Comment:
I think the Google C++ style guide is labeling an anonymous namespace as:

```cpp
}  // anonymous namespace
```

rather than:

```cpp
}  // namespace anonymous
```

or simply:

```cpp
}  // namespace
```

but I kinda like the empty `namespace` because it literally matches the declaration, so SGTM.